src/features/views/utils/getUniqueColumnName.ts

```diff
@@ -0,0 +1,20 @@
+import { ZetkinViewColumn } from '../components/types';
+
+export default function getUniqueColumnName(
+  defaultTitle: string,
+  columns: ZetkinViewColumn[]
+) {
+  let title = defaultTitle;
+  let count = 2;
+  let foundNewTitle = false;
+  while (!foundNewTitle) {
+    const titleIsAlreadyIn = columns.some((column) => column.title === title);
+    if (titleIsAlreadyIn) {
+      title = defaultTitle + ' ' + count.toString();
```

Comment:
is there a reason to use an "old school" string here, and not a template literal?

Reply:
Nice catch! I'll change it to template literal!